### DISCUSSION TITLE
Navbar language dropdown

### DISCUSSION THREAD

**Discussion by DaniBCD** [vuepress-reco/vuepress-theme-reco#281] [Q&A] [answered]
Hi, I can't figure out how to custom the language dropdown menu. In the example page, it alternates the text when changing the language, but mine doesn't do the same. Also, selectLanguageName seems to not be working. Anyone can help me or point me in the right direction? Thank you in advance

**Comment by recoluan** [comment, marked as answer]
You can refer to here https://ecosystem.vuejs.press/zh/themes/default/locale.html
